guides/gatherers.md

```diff
@@ -197,6 +198,55 @@ facts:
 
 For extra information refer to [trento-project/agent/../gatherers/sbd_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sbd_test.go)
 
+### sbd_dump
+
+This gatherer allows accessing the dumped data about all the configured devices on target nodes.
+It executes `sbd -d <device> dump` for all the devices in `/etc/sysconfig/sbd` and aggregates results in only one fact.
+
+Note that:
+- no arguments are required and any provided argument would just be ignored.
+- if any dump fails, a fact error is returned
+
+```rust
+{
+  "/path/to/a-device" => {
+    "header_version"   => 2.1,
+    "number_of_slots"  => 255,
+    "sector_size"      => 512,
+    "timeout_allocate" => 2,
+    "timeout_loop"     => 1,
+    "timeout_msgwait"  => 10,
+    "timeout_watchdog" => 5,
+    "uuid"             => "e09c8993-0cba-438d-a4c3-78e91f58ee52",
+  },
+  "/path/to/another-device" => {
+    "header_version"   => 2.1,
+    "number_of_slots"  => 255,
+    "sector_size"      => 512,
+    "timeout_allocate" => 2,
+    "timeout_loop"     => 1,
+    "timeout_msgwait"  => 10,
+    "timeout_watchdog" => 5,
+    "uuid"             => "e5b7c05a-1d3c-43d0-827a-9d4dd05ca54a",
+  }
+}
+```
+
+
+Specification examples:
+
+```yaml
+facts:
+  - name: sbd_devices_dump
+    gatherer: sbd_dump
+
+  - name: sbd_devices_dump
+    gatherer: sbd_dump
+    argument: whatever_argument # ignored
```

Comment:
I would remove the ignored argument, as the gatherer does not accept arguments and this is something it shouldn't be present anyway